packages/create-playwright/assets/examples/2-element-selectors.spec.ts

```diff
@@ -0,0 +1,37 @@
+
+import { test } from '@playwright/test';
+
+// Element selector overview, for more information see
+// https://playwright.dev/docs/selectors
+test('should be able to use selectors', async ({ page }) => {
+  await page.goto('/selectors.html');
+  // Text selector
+  await page.click('text=Log in');
+
+  // CSS selector
+  await page.click('button');
+  await page.click('#nav-bar .contact-us-item');
+
+  // Select by attribute, with css selector
+  await page.click('[data-test=login-button]');
+  await page.click('[aria-label="Sign in"]');
+
+  // Combine css and text selectors
+  await page.click('article:has-text("Playwright")');
+  await page.click('#nav-bar :text("Contact us")');
```

Comment:
#nav-bar >> text=Contact Us

Reply:
Hmm, this is taken from the official docs, shall we fix it there too? 
https://playwright.dev/docs/selectors

Reply:
Yes!